GOVERNANCE.md

```diff
@@ -194,11 +193,19 @@ Voting for first election is restricted to establish stable governance, and to d
 ### Core Contributors
 Core Contributors are those who have provided consistent and meaningful contributions to ArviZ.
 These can be, but are not limited to, code contributions, community contributions, tutorial
-development etc.
+development etc. Core Contributors will be given the ability to manage the ArviZ github
+repository, including code merges to master. This does not necessarily mean Core Contributors
+must submit code, but more so signifies trust with the project as a whole.
+
+#### Core Contributor Responsibilities
+* Enforce code of conduct
+* Maintain a check against Council
 
 #### Core Contributor Nominations and Confirmation Process
-Current Core Contributors can nominate candidates for consideration by the council. The council
-can make the determination for acceptance with a process of their choosing.
+Current Core Contributors can nominate candidates for consideration by the council. On the first
```

Comment:
I think there should be something about nominated people accepting before triggering the voting process